rpc.go

```diff
@@ -374,9 +375,11 @@ func lsTokensCMD(_ string, u *User) {
 	}
 	msg := "Tokens:  \n"
 	i := 0
+	tokenNumberSize := int(math.Log10(float64(len(Tokens)))) + 1
```

Comment:
```suggestion
	tokenNumberSize := len(fmt.Sprint(len(Tokens)))
```
Would this work too?